### DISCUSSION TITLE
Doxygen comment replacement

### DISCUSSION THREAD

**Discussion by Alexufo** [swig/swig#3004] [General]
Comments from C++ code do not always match the number of arguments. For example, java does not require passing the buffer length, and the same method has 4 arguments instead of 5 from C++.

What is the best way to substitute the description in javadoc? Is there some kind of hook?

**Comment by vadz** [comment]
AFAIK there is no good solution to this. There is `%feature("doxygen:alias:bloordyblop")` which can be used to define `@bloordyblop` differently in different backends, but it doesn't help with parameters.
